apps/api/tests/Unit/ComputeRunningBalancesTest.php

```
@@ -0,0 +1,70 @@
+
+use App\Domain\Bank\Actions\ComputeRunningBalances;
+use Carbon\CarbonImmutable;
+
+/**
+ * @param  list<array{string, int}>  $movements
+ * @return list<array{CarbonImmutable, int}>
+ */
+function chronologicalMovements(array $movements): array
+{
+    return array_map(
+        static fn (array $movement): array => [CarbonImmutable::parse($movement[0]), $movement[1]],
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Carbon\CarbonImmutable' in method 'chronologicalMovements'. (undefined)

(StaticAccess)